packages/riipen-ui/src/components/Menu.jsx

```diff
@@ -70,14 +70,21 @@ class Menu extends React.Component {
     /**
      * The index of the item selected in the list
      */
-    selectedIndex: PropTypes.number
+    selectedIndex: PropTypes.number,
+
+    /**
+     * The type of menu to create
+     */
+    type: PropTypes.oneOf(["menu", "selection"])
```

Comment:
We have been renaming this to `variant` instead of `type` in the UI kit. Better switch this for consistency.